src/components/menu/RowContextMenu.tsx

```diff
@@ -34,13 +34,30 @@ const RowContextMenu: React.FC<RowContextMenuProps> = ({}) => {
     return false;
   }
 
+  function onCopyCellContent(p: ItemParams) {
+    const { props } = p;
+    const { rowIdx } = props;
+    const row = state.rows[rowIdx];
+    const columnKey = state
+      .gridColumns[state.selectedCellPosition?.idx as number]
+      .key;
+
+    const value = row[columnKey];
+    const text = typeof value === 'object' 
```

Comment:
We should extract [formatClipboardValue](https://github.com/supabase/grid/blob/d90d95f44810af7b14b8374944ae9975b72a52da/src/components/common/Shortcuts.tsx#L88-L93) into `utils/common` and re-use it here.

Reply:
Got it